capture_service/dive_client_cli.cc

```diff
@@ -377,21 +457,25 @@ void trigger_gfxr_capture(Dive::DeviceManager& mgr,
                               << std::endl;
                     return;
                 }
+
+                // Retrieve the capture and asset file at the end of the capture.
+                retrieve_gfxr_capture(mgr, gfxr_capture_directory);
                 is_capturing = false;
-                std::cout << "Capture complete." << std::endl;
+                std::cout << capture_complete_message << std::endl;
             }
             else
             {
                 ret = mgr.GetDevice()->Adb().Run(
                 "shell setprop debug.gfxrecon.capture_android_trigger true");
+
                 if (!ret.ok())
                 {
                     std::cout << "There was an error starting the gfxr runtime capture."
                               << std::endl;
                     return;
                 }
                 is_capturing = true;
-                std::cout << "Capture started." << std::endl;
+                std::cout << "Capture started. Press g+enter to stop the capture." << std::endl;
             }
```

Comment:
sorry i am a bit confused here: I thought we only capture 1 frame?

Reply:
Yes, only a single frame is captured in each capture file. The second g+enter is used to set capture_android_trigger to false and retrieve the capture from the device. I can reword it so that it is "Press g+enter to retrieve the capture" instead

Reply:
yeah, i think that would be better. also need to update the comments in line 429